NEWS.md

```diff
@@ -1,3 +1,10 @@
+# teal.reporter 0.1.1.9000
+
+### New features
+
+* When adding a card, the cursor is focused in the comment block.
+* Report Previewer renders markdown as in downloaded document.
```

Comment:
According to our good practice, news entries are preferably in the following format:
"past verb + statement", for example: